server/app/constants/common.ts

```diff
@@ -1,3 +1,5 @@
 const NON_AUTH_PATHS: string[] = ['/user/login', '/user/register'];
 
-export { NON_AUTH_PATHS };
+const NON_GENERAL_PATHS: string[] = ['/user'];
```

Comment:
I'm a little confused, what are non-general paths?

Reply:
oh true the name sucks, but its all the paths that a general user can't make requests to.

so they can get a list of bikes but they cant get a list of users

Reply:
fixed name